src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java

```diff
@@ -227,23 +232,37 @@ public void onAttemptBreakSign(BlockBreakEvent event) {
     }
 
     //protect sign from being changed
-    @EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
-    public void onAttemptChangeLockerSign(SignChangeEvent event){
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void onAttemptChangeLockerSign(SignChangeEvent event) {
         Block block = event.getBlock();
-        if(LocketteProAPI.isLockSign(block)||LocketteProAPI.isAdditionalSign(block)){
-            Sign sign = (Sign)block.getState();
+        if (LocketteProAPI.isLockSign(block) || LocketteProAPI.isAdditionalSign(block)) {
+            Sign sign = (Sign) block.getState();
             sign.setWaxed(true);
             sign.update();
             event.setCancelled(true);
+
```

Comment:
Then just remove it ^^

Reply:
I left it as a comment in, so in future merges it doesn't crawl back out of it's grave